Sample Application/SampleApplication/SampleApplication.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.security.app-sandbox</key>
+    <true/>
+    <key>com.apple.security.files.user-selected.read-only</key>
+    <true/>
```

Comment:
What are these permissions for?

Reply:
Default permissions that comes with any new project 👍

Reply:
Seems to be [MacOS entitlements](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_files_user-selected_read-only). Suppose they don't hurt either way, but then the project was originally created targeting Mac directly instead of iPhone -> iPad -> Mac (designed for iPad)? 🤔